.hound.yml

```diff
@@ -0,0 +1,16 @@
+scss:
+  enabled: false
```

Comment:
Why enable css but not scss?

Reply:
one is the default scss (scss-lint) vs the other (stylelint) they are different tools.